src/main/java/com/mcmoddev/alt/util/ALTFileUtils.java

```diff
@@ -113,4 +120,77 @@ public static void safeDeleteDirectory(File file) {
 		}
 	}
 
+	public static void copyFromResourceIfNotPresent(ResourceLocation value) {
+		Path base = Paths.get(AdditionalLootTables.getLootFolder().toString(), value.getResourceDomain()).normalize();
+		createDirectoryIfNotPresent( base );
+		File container = Loader.instance().getIndexedModList().get(value.getResourceDomain()).getSource();

```

Comment:
Needs to handle the null return from indexed mod map not containing the modid. Should log if null.